ds701_book/24-NN-II-Backprop.qmd

```diff
@@ -145,79 +120,84 @@ plt.grid()
 
 There are many other variations. See for example [PyTorch Non-linear Activations](https://pytorch.org/docs/stable/nn.html#non-linear-activations-weighted-sum-nonlinearity)
 
-### Relating Back to _Another_ Earlier Lecture
+## Relating Back to an Earlier Lecture
 
 __Question__
 
 What does 
 
-$$ \mathrm{sigmoid}(\sum_i w_i x_i + b) \hspace{10pt} \textrm{where} \hspace{10pt} \mathrm{sigmoid}(x) = \frac{1}{1 + e^{-x}}$$
+$$
+\mathrm{sigmoid}(\sum_i w_i x_i + b) \hspace{10pt} \textrm{where} \hspace{10pt} \mathrm{sigmoid}(x) = \frac{1}{1 + e^{-x}}
```

Comment:
Use \left( and \right) around the sum to make the parentheses match the size of the sum.